src/main/java/uk/nhs/digital/nhsconnect/lab/results/utils/JmsHeaders.java

```diff
@@ -0,0 +1,7 @@
+package uk.nhs.digital.nhsconnect.lab.results.utils;
+
+public final class JmsHeaders {
```

Comment:
can this class be removed?

Reply:
Yeah it's not currently being used anywhere so I've removed it